deps.edn

```diff
@@ -16,6 +16,7 @@
   aero/aero                                {:mvn/version "1.1.6"}
   selmer/selmer                            {:mvn/version "1.12.59"}
   ch.qos.logback/logback-classic           {:mvn/version "1.3.14"}
+  commons-io/commons-io                    {:mvn/version "2.14.0"}
```

Comment:
Is there not an exclusion somewhere?

Reply:
Don't need it. If you pin a top-level dep, you'll note all the conflicting subdeps in the tree will say `:use-top` and not install

Reply:
They were much more necessary the way Lein did deps. I think there are still some good cases where we can use them but it just isn't important here.